protocol_driver_grpc.cc

```diff
@@ -198,4 +196,7 @@ void ProtocolDriverGrpc::ShutdownServer() {
   }
 }
 
+void ProtocolDriverGrpc::ShutdownServer() {
+}
```

Comment:
I am doubting that this should be an empty function. Especially now that we are launching threads for the incoming RPC action lisst from inside the protocol driver.

Reply:
Added code to call Shutdown on the GRPC server